src/serviceWorker.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
Might want to keep this so ESLint ignores the contents of this file. It's not usually modified from what I know